src/main/scala/de/heikoseeberger/sbtheader/license/License.scala

```diff
@@ -76,3 +76,65 @@ object MIT extends License {
         |""".stripMargin
   }
 }
+
+object BSD2Clause extends License {
+  override def createLicenseText(yyyy: String, copyrightOwner: String) = {
+    s"""|Copyright (c) $yyyy, $copyrightOwner
+        |All rights reserved.
+        |
+        |Redistribution and use in source and binary forms, with or without modification,
+        |are permitted provided that the following conditions are met:
+        |
+        |1. Redistributions of source code must retain the above copyright notice, this
+        |list of conditions and the following disclaimer.
```

Comment:
are you ok with this lack of word wrapping on the paragraph? This looks like

```
1. Redistributions of source code must retain the above copyright notice, this
list of conditions and the following disclaimer.
```

whereas many people may prefer

```
1. Redistributions of source code must retain the above copyright notice, this
    list of conditions and the following disclaimer.
```